File: LLM-Translations/GPT-4/codenet/Python/Go/s590045523.go

package main

import (
	"fmt"
	"strconv"
	"strings"
)

func main() {
	var N int
	fmt.Scan(&N)
	var input string
	fmt.Scan(&input)

	Lstr := strings.Split(input, " ")
	L := make([]int, N)
	for i, s := range Lstr {
		L[i], _ = strconv.Atoi(s)
	}

	ans := 0
	for i := 0; i < N-2; i++ {
		for j := i + 1; j < N-1; j++ {
			for k := j + 1; k < N; k++ {
				a := L[i]
				b := L[j]
				c := L[k]
				if a == b || b == c || c == a {
					continue
				}
				if a+b > c && b+c > a && c+a > b {
					ans++
				}
			}
		}
	}
	fmt.Println(ans)
}
// 

File: LLM-Translations/GPT-4/codenet/Python/C/s590045523.c
#include <stdio.h>

int main() {
    int N, ans = 0;
    scanf("%d", &N);
    int L[N];

    for(int i = 0; i < N; i++) {
        scanf("%d", &L[i]);
    }

    for(int i = 0; i < N - 2; i++) {
        for(int j = i + 1; j < N - 1; j++) {
            for(int k = j + 1; k < N; k++) {
                int a = L[i];
                int b = L[j];
                int c = L[k];

                if(a == b || b == c || c == a) {
                    continue;
                }

                if(a + b > c && b + c > a && c + a > b) {
                    ans++;
                }
            }
        }
    }

    printf("%d\n", ans);
    return 0;
}

// 
